pom.xml

```diff
@@ -1,341 +1,398 @@
-                            <outputDirectory>${basedir}/wiki/resources/images</outputDirectory>
-                            <resources>
-                                <resource>
-                                    <directory>${basedir}/wiki/images</directory>
-                                </resource>
-                            </resources>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-site-plugin</artifactId>
+				<version>3.7.1</version>
+				<configuration>
+					<!-- This is where site.xml is located -->
+					<siteDirectory>${basedir}/wiki</siteDirectory>
+					<!-- Github pages only generates documentation from docs directory -->
+					<outputDirectory>${basedir}/docs</outputDirectory>
+				</configuration>
+				<dependencies>
+					<dependency>
+						<groupId>org.apache.maven.doxia</groupId>
+						<artifactId>doxia-module-markdown</artifactId>
+						<version>1.8</version>
+					</dependency>
+				</dependencies>
+			</plugin>
 
-            <plugin>
-                <artifactId>maven-clean-plugin</artifactId>
-                <version>3.1.0</version>
-                <!-- We want to remove duplicate/copied resources from site
-                    generation -->
-                <configuration>
-                    <filesets>
-                        <fileset>
-                            <directory>${basedir}/wiki/resources</directory>
-                        </fileset>
-                    </filesets>
-                </configuration>
-            </plugin>
+			<plugin>
+				<artifactId>maven-resources-plugin</artifactId>
+				<version>3.1.0</version>
+				<executions>
+					<execution>
+						<id>copy-resources</id>
+						<!-- here the phase you need -->
+						<phase>site</phase>
+						<goals>
+							<goal>copy-resources</goal>
+						</goals>
+						<configuration>
+							<!-- plugin requires all resources to exist within resources/ directory -->
+							<!-- but we don't want to break links in documentation -->
+							<outputDirectory>${basedir}/wiki/resources/images</outputDirectory>
+							<resources>
+								<resource>
+									<directory>${basedir}/wiki/images</directory>
+								</resource>
+							</resources>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
 
-        </plugins>
-    </build>
+			<plugin>
+				<artifactId>maven-clean-plugin</artifactId>
+				<version>3.1.0</version>
+				<!-- We want to remove duplicate/copied resources from site generation -->
+				<configuration>
+					<filesets>
+						<fileset>
+							<directory>${basedir}/wiki/resources</directory>
+						</fileset>
+					</filesets>
+				</configuration>
+			</plugin>
+
+		</plugins>
+	</build>
+	<profiles>
+		<!-- The Configuration of the integration-test profile -->
+		<profile>
+			<id>integrationTest</id>
```

Comment:
the travi.yml needs to be updated with the new command with a profile.

Reply:
done